ui/ui.go

```diff
@@ -179,6 +186,7 @@ func (q queryHandler) ServeHTTP(writer http.ResponseWriter, request *http.Reques
 		Metadata: result.Metadata,
 		Name:     cmd.Name(),
 	}
+	serveHTTPTimerCompletion()
```

Comment:
It can't go any lower, since we want to include the profiler's results in `response` (assigned just below this).

`convertProfile` needs the profiler to have all its timers finished, and the profile data needs to be ready for being put in the `response` to be written (again, before the hook).